src/main/java/seedu/address/model/ingredient/Other.java

```diff
@@ -0,0 +1,17 @@
+package seedu.address.model.ingredient;
+
+/**
+ * Represents Other Ingredients in the recipe book. These ingredients include spices, liquids and condiments.
+ */
+
+public class Other extends Ingredient {
+
+    public Other(double quantity, String name) {
+        super(quantity, name);
+        super.unit = Unit.COUNTABLE;
```

Comment:
Just commenting: since "other" ingredients include spices, _**liquids**_, and condiments, we need to think of how to parse the "io/" input so that we can assign an appropriate unit to the ingredient (because as of now, we're only assigning its unit as Unit.COUNTABLE)